app/views/components/span.scala.html

```diff
@@ -23,6 +23,6 @@
    id: Option[String] = None
 )(implicit msgs: Messages)
 
- @cls = @{if(classes.isDefined) classes.get.concat(" govuk-body") else "govuk-body"}
+ @cls = @{if(classes.isDefined) classes.get else "govuk-body"}
```

Comment:
do we want a standardClasses bool here for concat? I don't know the use cases for span

Reply:
I removed that as you can't get rid of govuk-body otherwise.